src/assets/blns.json

```diff
@@ -0,0 +1,517 @@
+[
```

Comment:
I am thinking that maybe we don't need this file since you have the same array list in getTestInput.js file. Or you could keep this file and import this test inputs in getTestInput.js file instead of defining a duplicate test array in the test file. It't not a quite big deal and either way sounds good to me. The only thing is to make it easier to maintain those test cases.

Reply:
I only kept it in as a reference in case people want to look through it or we need it later. I wasn't able to get the file reading to work, that's why it is duplicated in an array.

Reply:
Got it. I would approve it for this PR. That make sense to me. But please wrote down this issue in somewhere in Wiki and I would take a look it in Spring break.